script/script.js

```diff
@@ -0,0 +1,326 @@
+        item = item.slice(9, 14);
+        oMoves.push(item);
+        //add it to the total moves
+        totalMoves.push(event.target);
+        //function
+      }
+      checkWin();
+      //ask player to play somewhere else if there is already something in the cell
+    } else alert("You can't play there!");
+  },
+
+  //player vs. computer game
+  computerMove(event) {
+    //human goes first/is Player X
+    if (currentPlayer === "X") {
+      //only allow to put X in cell if it's empty
+      if (!event.target.textContent) {
+        //update cell
+        event.target.textContent = "X";
+        //switch players
+        currentPlayer = "O";
+        playerTurn.innerHTML = "It's the " + player2.value + "'s turn";
+        //clean up & add this move to the player's array
+        let item = event.target;
+        item = item.outerHTML.toString();
+        item = item.slice(9, 14);
+        xMoves.push(item);
+        //add it to the total moves
+        totalMoves.push(event.target);
+        //check to see if there is a winning combo
+        checkWin();
+        //trigger computer move
+        setTimeout(function () {
+          gameBoard.computerMove(event);
+        }, 1500);
+      }
+      //ask player to play somewhere else if the cell is not empty
+      else alert("You can't play there!");
+    } else if (currentPlayer === "O") {
+      //choose a random number between 0 and 8
+      let cellNum = Math.floor(Math.random() * 9);
+      //turn this into an index to randomly select a cell for computer to play in
+      let compChoice = allCells[cellNum];
+      //only fill in the cell if it is "false" because it's empty
+      if (!compChoice.textContent) {
+        compChoice.textContent = "O";
+        //switch players
+        currentPlayer = "X";
+        playerTurn.innerHTML = "It's " + player1.value + "'s turn";
+        //clean up & add this move to the player's array
+        let item = compChoice;
+        item = item.outerHTML.toString();
+        item = item.slice(9, 14);
+        oMoves.push(item);
+        //add it to the total moves
+        totalMoves.push(event.target);
+        //check to see if there is a winning combo
+        checkWin();
+        //if there is already something in the cell, trigger the computer choice function again
+      } else if (compChoice.textContent) {
+        gameBoard.computerMove(event);
+      }
+    }
+  },
+};
+
+//all the winning combos
+let winningArrays = [
+  ["cell1", "cell2", "cell3"],
+  ["cell4", "cell5", "cell6"],
+  ["cell7", "cell8", "cell9"],
+  ["cell1", "cell4", "cell7"],
+  ["cell2", "cell5", "cell8"],
+  ["cell3", "cell6", "cell9"],
+  ["cell1", "cell5", "cell9"],
+  ["cell3", "cell5", "cell7"],
+];
+
+function checkWin() {
+  //check to see if the player has a winning combination - if they do, then highlight the winning cells, set an alert, and stop the timer
+  //loop through the arrays
+  for (let i = 0; i < winningArrays.length; i++) {
```

Comment:
Using this style of standard for loop is not generally recommended. It is confusing to read, and there are almost always better styles of for (or even while) loops to use in place.